index.js

```diff
@@ -80,6 +80,13 @@ class Client {
     socket.setTimeout(options.timeout || this.timeout)
 
     socket
+      .on('timeout', () => {
+        socket.end()
+        const err = new Error('Request timed out. Hostname: ', this.hostname)
```

Comment:
I don't think this will work, I'm pretty sure `Error` only takes one parameter.

Reply:
Looks like the second parameter is supposed to be used for the filename?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error

Reply:
You're right, whoops, fixing it here: https://github.com/segmentio/jsonrpc2.js/pull/27/files